ignite/distributed/comp_models/base.py

```diff
@@ -96,7 +96,7 @@ def _encode_str(x: str, device: torch.device, size: int) -> torch.Tensor:
 
     def _get_max_length(self, x: str, device: torch.device) -> int:
         size = torch.tensor([len(x)], device=device)
-        size = self._do_all_reduce(size, "MAX")
+        size = self._do_all_reduce(size, group=None, op="MAX")
```

Comment:
Let's make `group` arg default as None, such that you do not need to update this code